composer.json

```diff
@@ -23,15 +23,15 @@
   ],
   "require": {
     "php": ">=7.1",
-    "fzaninotto/faker": "^1.4",
-    "intervention/image": "^2.4"
+    "fakerphp/faker": "^1.13.0",
+    "intervention/image": "^2.5.1"
```

Comment:
It does not make sense because you can update for newer version with this value
https://getcomposer.org/doc/articles/versions.md#caret-version-range-